### ISSUE TITLE
Accessing plyr events

### ISSUE THREAD

**Issue by ricardoalcocer** [biati-digital/glightbox#123]
Is there an easy way of listening to events triggered by plyr on a per-video basis?  I need to track ready, play, pause, ended, timeupdate, and perhaps others, but these events will be slightly different for each video.

I see it's being instantiated here: https://github.com/biati-digital/glightbox/blob/325f6d7aa094fe67d6d7cb7d2a92595c043b42dc/dist/js/glightbox.js#L1303, but how can I access it from code?

Thank you for a great project!

**Comment by biati-digital** [comment]
There's no way to access that for now, I could probably pass the Plyr instance on the methods **afterSlideChange** and **beforeSlideChange** for me it makes sense to add them here as afterSlideChange is triggered when a slide becomes active.

```javascript
afterSlideChange: (active) => {
    const {index, slide, player} = active;

    // player will contain the Plyr instance
    player.on('pause', event => {
        // do something
    });
}
```

I'll work on this as soon as I have some free time.

**Comment by ricardoalcocer** [comment]
Awesome @biati-digital 
That makes sense to me too 

Gracias!

**Comment by melvas6** [comment]
I too have a similar need to know if the end of video is reached.
Ability to trigger plyr event would be helpful.
Thanks for this beautiful project.

**Comment by jschelkens** [comment]
This is also something I really could use at the moment (need to track when the video starts,  >= 25%,  >= 50%,  >= 75%, complete). Great project!

**Comment by biati-digital** [comment]
This has been implemented in the latest version 3.0.0, please check the [Events Section](https://github.com/biati-digital/glightbox#events) in the Readme, there's an example of how to access the player instance. Also there are two new methods called **getSlidePlayerInstance** and **getAllPlayers** check the Readme for more info.

**Comment by biati-digital** [comment]
Closing this as it has been solved.
